scipy/stats/morestats.py

```diff
@@ -2722,6 +2722,20 @@ def circmean(samples, high=2*pi, low=0, axis=None):
     -------
     circmean : float
         Circular mean.
+        
+    Examples
+    --------
+    
+    >>> from scipy.stats import morestats
+    >>> import numpy as np
```

Comment:
This import is implicit in all scipy documentation, best remove it here and elsewhere.